app/src/main/java/com/iic/shopingo/data/Location.java

```diff
@@ -0,0 +1,61 @@
+package com.iic.shopingo.data;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by asafg on 03/03/15.
+ */
+public class Location implements Parcelable{
```

Comment:
That's exactly the thing the code formatter solves so we won't have to deal with it in the pr.